README.md

```diff
@@ -1,80 +1,138 @@
+Morphdom is a lightweight module for manipulating your HTML DOM tree to match a
+given HTML string, DOM or Virtual DOM input. Morphdom figures out the minimal
+number of operations to achieve this.
 
-When using this mixin, each call to render will patch your view, make it always up to date if render is called after each state change.
+This mixin lets you use morphdom with Marionette without changing your template
+engine. The morphdom mixin patches `render` to update the view instead of using
+`innerHTML`.
 
+## Using marionette.morphdom
 
-# Install
+To use marionette.morphdom in your code, inject `morphdomMixin` into your `View`:
+
+```js
+import Mn from 'backbone.marionette';
+import morphdomMixin from 'marionette.morphdom';
+
+// Note the morphdomMixin inside the extend chain
+export const Mn.View.extend(morphdomMixin).extend({
+  // Your regular code here
+})
+```
+
+### Install
+
+`marionette.morphdom` is available on NPM:
 
 ```
 npm i marionette.morphdom
 ```
 
 
-# Example
+### Example
+
+To use the morphdom mixin, just import from `marionette.morphdom` and call
+`Mn.View.extend` with the morphdom mixin. An example below:
 
 ```js
-import Backone from 'backbone';
+import Bb from 'backbone';
 import Mn from 'backbone.marionette';
 import morphdomMixin from 'marionette.morphdom';
 
-const MyView = Mn.View.extend(morphdomMixin).extend({
-    template: data => `<section><h1>${data.title}</h1><p>${data.article}</p></section>`,
+export const MyView = Mn.View.extend(morphdomMixin).extend({
+  template: data => `<section><h1>${data.title}</h1><p>${data.article}</p></section>`,
 
-    modelEvents: {
-        change: 'render'
-    },
+  modelEvents: {
+    change: 'render'
+  },
 
-    initialize: function() {
-        this.model = new Backbone.Model({
-            title: 'How to keep this view update to date?',
-            article: 'You should use marionette morphdom!'
-        });
-    }
+  initialize: function() {
+    this.model = new Bb.Model({
+      title: 'How do I keep this view update to date?',
+      article: 'Use marionette morphdom!'
+    });
+  }
 
 });
-
-export default MyView;
 ```
 
-That's it. Anytime your model change, your view will be patched.
+That's it. Anytime your model change, morphdom will take over the `render` for
+you.
 
 
-# FAQ
+## FAQ
 
-## Why?
+### Why?
 
 Because default Marionette render does `innerHTML` to update the view.
 
-## Is this slower or faster than... ?
+### Is this slower or faster than... ?
+
+For most Marionette users who just use `render` for updating your views, this
+will be a huge performance boost.
+
+If, however, you perform a lot of direct DOM manipulation or use data binding
+tools like `backbone.stickit`, then this may be slower. Morpdom must render the
+entire view before it can patch it. Depending on the size of your views, this
```

Comment:
Nope, marionette.morphdom. This plugin call to render, but morphdom just do patch.

Morphdom (and you miss the *h*) is the lib under it. I must be careful not to confuse people marionette.morphdom !== morphdom